### DISCUSSION TITLE
if I do ;bal 200k first then ;bal 300k is not working . so can you tell me a way in which both of them work

### DISCUSSION THREAD

**Discussion by Ansihv2008** [Rapptz/discord.py#7218] [Q&A]
@client.command()
@commands.has_role('Auction boi')
async def auction(ctx, *, data ,  mssg=None):
    data = data.split(',') 
    embed = discord.Embed(
        title= ctx.author.display_name , description= "Welcome to Poke auc all day auctions", color=0x00ff00)
    embed.add_field(name = "Pokemon" , value = f"{data[0]}" , inline = False)
    embed.add_field(name = "Autobuy" , value = f"{data[1]}" , inline = False)
    embed.add_field(name = "Time" , value = f"{data[2]}" , inline = False)
    embed.add_field(name = "Highest bidder" , value = f"{data[3]}" , inline = False)
    embed.add_field(name = "Amount" , value = f"{data[4]}" , inline = False)
    await ctx.send(embed=embed)
    def check(msg):
        return msg.author == ctx.author and msg.channel == ctx.channel and \
        msg.content.lower() in [";bal 200k", ";bal 1m" , ";bal 300k"]

    msg = await client.wait_for("message", check=check)
    if msg.content.lower() == ";bal 200k":
        await ctx.send((embed.set_field_at(4, name = "Amount", value = "200k")))
    elif msg.content.lower() == ";bal 300k":
        await ctx.send((embed.set_field_at(4, name = "Amount", value = "300k")))
    else:
        await ctx.send("You said no!")
    await ctx.send(embed=embed)

                                                                                                                                                  if I do ;bal 200k first then ;bal 300k is not working . so can you tell me a way in which both of them work

**Comment by theAbdoSabbagh** [comment]
if you mean it responds to ";bal 200k" but doesn't respond to ";bal 300k" **afterwards**, without executing the command again, its because the command waits for the message once, it responds to it, then it finishes. If you want your bot to also respond to ";bal 300k" without executing the command again, I suggest using a for loop.
